iree/compiler/Codegen/LLVMGPU/ConvertToLLVM.cpp

```diff
@@ -336,36 +336,18 @@ class ConvertIREEConstantOp : public ConvertToLLVMPattern {
 };
 
 /// A pattern to convert hal.interface.workgroup.id/count/size into
-/// corresponding NVVM/ROCDL ops.
-template <typename InterfaceOpTy, typename XOp, typename YOp, typename ZOp>
+/// corresponding GPU ops.
+template <typename InterfaceOpTy, typename NewOp>
```

Comment:
Nit: `NewOpTy` to be consistent with `InterfaceOpTy`?

Reply:
Done